src/vivarium_public_health/risks/effect.py

```diff
@@ -468,10 +469,15 @@ def get_target_modifier(
     ) -> Callable[[pd.Index, pd.Series], pd.Series]:
         def adjust_target(index: pd.Index, target: pd.Series) -> pd.Series:
             rr_intervals = self.lookup_tables["relative_risk"](index)
-            exposure = self.exposure(index)
-            # use exposures and rr points to calculate m and b in y=mx+b
-            # ie m = (y2-y1)/(x2-x1) and b = y1 - mx1
-            # plug exposures into mx+b to get our RRs
+            exposure = self.population_view.get(index)[f"{self.risk.name}_exposure"]
+            x1, x2 = (
+                rr_intervals["left_exposure"].values,
+                rr_intervals["right_exposure"].values,
+            )
+            y1, y2 = rr_intervals["left_rr"].values, rr_intervals["right_rr"].values
+            m = (y2 - y1) / (x2 - x1)
```

Comment:
Is it possible to have x2 = x1 and get a DivideByZero?

Reply:
Good question. x1 is the largest exposure for which an RR is defined that is less than or equal to a simulant's exposure and x2 is the smallest exposure for which an RR is defined that is greater than simulant's exposure, so they should never be equal.